kuma/static/js/components/payments/focus-visible.js

```diff
@@ -0,0 +1,27 @@
+(function() {
+    'use strict';
+    var contributeForm = document.getElementById('contribute-form');
+
+    function removeFocusVisible() {
+        this.classList.remove('focus-visible');
+        this.removeEventListener('focusout', removeFocusVisible);
+    }
+
+    // When a keyup event is triggered on the contribute form,
+    contributeForm.addEventListener('keyup', function(event) {
+        var currentTarget = event.target;
```

Comment:
What happens if the user clicks on text field #1 and the types the tab key to move focus to text field #2. Which one gets the focus ring?  I'd guess that the focus and blur events happen between the keydown and keyup events, so that by the time the keyup event arrives focus has moved. But I'd also expect that event.target would have to be the same for keydown and key up, which would imply that the wrong text field would get the focus ring in that case.

So would it be better to use document.activeElement here instead of event.target?